cmd/config/get/get.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package get
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/config"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+var (
+	Writer io.Writer = os.Stdout
+	args   struct {
+		debug bool
+	}
+)
+
+var Cmd = NewConfigCommand()
+
+func NewConfigCommand() *cobra.Command {
+	Cmd := &cobra.Command{
+		Use:   "get [flags] VARIABLE",
+		Short: "Prints the value of a config variable",
+		Long:  "Prints the value of a config variable. See 'rosa config --help' for supported config variables.",
+		Args:  cobra.ExactArgs(1),
+		Run:   run,
+	}
+	flags := Cmd.Flags()
+	flags.BoolVar(
+		&args.debug,
+		"debug",
+		false,
+		"Enable debug mode.",
+	)
+	return Cmd
+}
+
+func run(cmd *cobra.Command, argv []string) {
+	r := rosa.NewRuntime()
+
+	err := PrintConfig(argv[0])
+	if err != nil {
+		r.Reporter.Errorf(err.Error())
+		os.Exit(1)
+	}
+}
+
+func PrintConfig(arg string) error {
+	// Load the configuration file:
+	cfg, err := config.Load()
+	if err != nil {
+		err := fmt.Errorf("Can't load config file: %v", err)
```

Comment:
@tirthct Can be shortened to just `return fmt.Errorf("Failed to load config file: %v", err)`

Reply:
Changed